dev-client/package-lock.json

```diff
@@ -98,7 +98,6 @@
         "react-devtools": "^5.3.1",
         "react-native-svg-transformer": "^1.5.0",
         "react-test-renderer": "^18.3.1",
-        "ts-jest": "^29.2.5",
```

Comment:
issue: you removed this file by accident.

Reply:
not an accident, it's no longer used

Reply:
sorry for the mixup. GitHub's UI made it looked like to remove the whole file.